README.md

```diff
@@ -27,7 +27,7 @@ Next set this project up with:
 
 Now copy the required files to this service:
 ~~~
- $ mkdir ./data/certs
+ $ mkdir -p ./data/certs
```

Comment:
+1

Reply:
I just cherry-picked this into main. https://github.com/foundriesio/factory-registration-ref/commit/c05be99da3d4878d562cee5ebf462d52be17f15e